SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security Policy
+
+The Uptane community is committed to maintaining a reliable and consistent Standard. If you believe you have identified errata in the Uptane Standard, please follow these guidelines for responsible disclosure.
+
+## Supported Versions
+
+We release updates to the Uptane specification to address errata. The versions eligible to receive updates are listed below:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.0.0   | :white_check_mark: |
+| 1.2.0   | :white_check_mark: |
+| 1.1.0   | :white_check_mark: |
+| 1.0.1   | :white_check_mark: |
+| 1.0.0   | :white_check_mark: |
+
+## Reporting Errata
+
+
+Please report (suspected) errata in the specification to `uptane-security [AT] googlegroups [DOT] com`. You will receive an acknowledgement from us within 72 hours. Keep in mind that Uptane is a community based project of volunteers and, therefore, response times could vary. If the issue is confirmed, we will provide a response within 90 days.
```

Comment:
Do we need a separate mailing list for this rather than just using the GitHub issue tracker? We decided not to make a private process, so we can probably use the existing uptane-standards mailing list if the issue tracker is not sufficient